lib/disability_compensation/providers/intent_to_file/lighthouse_intent_to_file_provider.rb

```diff
@@ -9,19 +9,18 @@ class LighthouseIntentToFileProvider
 
   def initialize(current_user)
     icn = current_user.icn
-    @service = BenefitsClaims::Service.new(icn)
+    ssn = current_user.ssn
+    @service = BenefitsClaims::Service.new(icn, ssn)
   end
 
   def get_intent_to_file(type, lighthouse_client_id, lighthouse_rsa_key_path)
     data = @service.get_intent_to_file(type, lighthouse_client_id, lighthouse_rsa_key_path)['data']
     transform(data)
   end
 
-  def create_intent_to_file(type)
-    # Will implement in 57064
-    # data = @service.get_intent_to_file(type)['data']
-    # return 401 response if something is missing?
-    # transform(data)
+  def create_intent_to_file(type, lighthouse_client_id, lighthouse_rsa_key_path)
+    data = @service.create_intent_to_file(type, lighthouse_client_id, lighthouse_rsa_key_path)['data']
+    transform(data)
```

Comment:
the `get` and the `create` methods look awfully similar.
wondering if there is a way to use the `.call(:#method_name)` here.
this isn't a big deal, just wondering out loud about ruby syntactical sugar :yum:

Reply:
so the method would look something like
```
private 

def intent_to_file_data(method_name, type, lighthouse_client_id, lighthouse_rsa_key_path)
  data = @service.send(method_name.to_sym, type, lighthouse_client_id, lighthouse_rsa_key_path)['data']
  transform(data)
end
 ```

however, please ignore this if we decide to send `ssn` into the `create` method.

and again, this would be syntactical sugar